packages/cli/src/commands/deploy/args.ts

```diff
@@ -98,9 +99,9 @@ export const args = {
   '--with-cache': Boolean,
   '--public': Boolean,
   '--no-clipboard': Boolean,
-  '--env': [String],
-  '--build-env': [String],
-  '--meta': [String],
+  '--env': String,
+  '--build-env': String,
+  '--meta': String,
```

Comment:
These need to remain arrays because the user can pass the flag multiple times. For example:

```
vercel  --env TAG=CANARY --env VERSION=1
```